LEGO1/lodlist.h

```diff
@@ -0,0 +1,180 @@
+#ifndef LODLIST_H
+#define LODLIST_H
+
+#include "assert.h"
+
+#include <stddef.h> // size_t
+
+class LODObject;
+
+// disable: identifier was truncated to '255' characters in the debug information
+#pragma warning(disable : 4786)
```

Comment:
I don't see this warning with msvc 4.2.
Where and how is it triggered?

Reply:
Templated names have the potential to generate this error.  A normal name plus namespaces generally wouldn't get close. At any rate, it was in the reference source file, so I left it in, but we may decided to clean up pragma's at some stage if we find them to be unnecessary.

Reply:
Thanks. When I do my next bi-monthly mingw check-up, I'll probably `#ifdef _MSC_VER` this out :)

Reply:
Yep, it's one of those strange issues that's apparently unique to MSVC.